tool/site/client-react/src/components/prescription-tracker/prescriptionDetails.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react"
+import {SummaryList} from "nhsuk-react-components"
+import {PrescriptionProps} from "./prescription"
+import {Task} from "fhir/r4"
+import {formatNhsNumber} from "../../formatters/demographics"
+import {formatDate} from "../../formatters/dates"
+import {getCourseOfTherapyTypeExtension} from "../../fhir/customExtensions"
+
+export interface PrescriptionDetailProps {
+  id: string
+  type: string
+  patientNhsNumber: string
+  creationDate: string
+  status: string
+}
+
+export function createPrescriptionDetailProps(task: Task): PrescriptionDetailProps {
+  const prescriptionType = getCourseOfTherapyTypeExtension()(task.extension).valueCoding.code
+  return {
+    id: task.focus.identifier.value,
+    type: prescriptionType.substring(0, 1).toUpperCase() + prescriptionType.substring(1),
```

Comment:
would be nice to use css `capitalise` style (maybe using styled jsx). if not, would be nice to extract this into a `Capitalise` formatter for other components to use

Reply:
infact, it would be nicer to access display instead of code here. we've made use of a courseOfTherapyType valueSet in a different PR so maybe we can implement that here when merged?